### ISSUE TITLE
Button color contrast

### ISSUE THREAD

**Issue by silentnoodlemaster** [generoi/sage#6]
shouldn't this be either `$button-color` or `$button-color-alt` depending on contrast?
genero.fi has a contrast ratio of 1.94 if the button is primary color. should be 4.5 for text smaller than x-large (18pt/24px)

https://github.com/generoi/sage/blob/d15973f97c18312f278b8c069addcadc7303cc6b/resources/assets/styles/components/foundation/_button.scss#L23-L29

new issue because i don't have time to figure it out

**Comment by silentnoodlemaster** [comment]
relevant WCAG: https://www.w3.org/WAI/WCAG21/quickref/?levels=aaa#contrast-minimum

**Comment by oxyc** [comment]
True. We might also entirely remove Foundations button markup and use their mixins directly here. That would fix this as well.

**Comment by oxyc** [comment]
I guess we do have to keep it actually. Didn't think that through. There's https://github.com/zurb/foundation-sites/blob/39056a7e821e052c1f7026505a4bbb748996d838/scss/util/_color.scss#L65

**Comment by silentnoodlemaster** [comment]
in this specific case base color can be any color
`button--primary`, `button--secondary` ...

**Comment by silentnoodlemaster** [comment]
oh right, it could be re-written to loop trough the palette 🤦‍♂️

**Comment by oxyc** [comment]
What do you think about just scrapping the default classes foundation generates for eg. buttons and add them here manually?

```scss
.button {
  @include button;
}
```

It's pretty rare that designers use more than 2 button styles and I find myself usually fighting against the default classes more than if I just defined them myself.

**Comment by silentnoodlemaster** [comment]
irrelevant after tailwind is added. #8 closes
